roles/kube_proxy/files/kube-proxy-images.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStreamTag
+metadata:
+  name: node:v3.10
+  namespace: kube-proxy
+tag:
+  reference: true
+  from:
+    kind: DockerImage
+    name: openshift/node:v3.10.0
```

Comment:
Its best not to hardcode version instead something like this should do the trick s/v3.10/${version}/

Reply:
It would be updated in https://github.com/openshift/openshift-ansible/pull/9621/files#diff-eaf9b159b10b04891c4d8f55d2352accR21